_sass/blocks/revenues/_revenues_subpage_diving_graphics.scss

```diff
@@ -196,113 +191,117 @@
         z-index: 2;
       }
     }
-  }
-
-  &.coal{
 
-    background: $coal-primary-color;
+    background: $offshore-renewables-primary-color;
 
-    .revenues_subpage-dyk_heading {
-      background-color: $coal-secondary-color;
+    .revenues_subpage-dyk_heading, {
+      background-color: $offshore-renewables-secondary-color;
     }
+
     .revenues_subpage-steps_heading,
     .revenues_subpage-steps_heading_more {
-      color: $coal-secondary-color;
+      color: $offshore-renewables-secondary-color;
     }
+  }
 
+  &.coal {
     @include respond-to(medium-up) {
       background: url('../img/revenues/coal-top.png') top left no-repeat, url('../img/revenues/coal-bottom.png') bottom left no-repeat;
       background-size: 100%, 100%;
-      position: relative;
       height: 100%;
+      position: relative;
       z-index: 2;
+
       &:before {
+        background: url('../img/revenues/coal-middle.png') top left repeat-y;
+        background-size: 100%;
+        bottom: 603px;
         content: '';
-        z-index: -1;
-        right: 0;
         left: 0;
-        top: 377px;
-        bottom: 603px;
         position: absolute;
-        background: url('../img/revenues/coal-middle.png') top left repeat-y;
-        background-size: 100%;
+        right: 0;
+        top: 377px;
+        z-index: -1;
       }
-
     }
 
     @include respond-to(large-up) {
 
       &:before {
-        top: 486px;
         bottom: 777px;
+        top: 486px;
       }
     }
 
     @include respond-to(huge-up) {
       &:before {
-        top: 503px;
         bottom: 804px;
+        top: 503px;
       }
     }
 
     @include respond-to(jumbo-up) {
       &:before {
-        top: 737px;
         bottom: 1178px;
+        top: 737px;
       }
     }
+
     @include respond-to(massive-up) {
       background: url('../img/revenues/coal-wide.png') no-repeat;
-      background-size: 185%;
       background-position-x: -500px;
+      background-size: 185%;
+
       &:before {
         background: none;
       }
     }
-  }
 
-  &.minerals{
-    background: $minerals-primary-color;
+    background: $coal-primary-color;
```

Comment:
Expected item on line 260 to appear before line 208. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets